{"name": "反向用例: raise TypeError", "status": "passed", "steps": [{"name": "调用相加", "status": "broken", "statusDetails": {"message": "TypeError: unsupported operand type(s) for +: 'int' and 'str'\n", "trace": "  File \"/Users/linhao/PycharmProjects/pythonProject/homework/venv_new/lib/python3.9/site-packages/allure_commons/_allure.py\", line 172, in impl\n    return func(*a, **kw)\n  File \"/Users/linhao/PycharmProjects/pythonProject/homework/Test_2021_06_17/calculator/calculator.py\", line 7, in add\n    return a + b\n"}, "parameters": [{"name": "a", "value": "1"}, {"name": "b", "value": "'2'"}], "start": 1624203709614, "stop": 1624203709614}], "attachments": [{"name": "log", "source": "23c3410c-b32f-4c64-9d68-1a49b1f689bf-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "a", "value": "1"}, {"name": "b", "value": "'2'"}, {"name": "error_type", "value": "'TypeError'"}], "start": 1624203709613, "stop": 1624203709615, "uuid": "bed2ecb2-5f32-4278-95e2-3a55de1f6f19", "historyId": "a6ee0ce94658cc2088a0ab0fdd923302", "testCaseId": "1b5aab2181c21923f6bd8223ee598e23", "fullName": "Test_2021_06_17.test_cal.test_cal.TestAddDivision#test_add_fail", "labels": [{"name": "feature", "value": "测试计算器"}, {"name": "story", "value": "测试加法"}, {"name": "parentSuite", "value": "Test_2021_06_17.test_cal"}, {"name": "suite", "value": "test_cal"}, {"name": "subSuite", "value": "TestAddDivision"}, {"name": "host", "value": "192.168.8.102"}, {"name": "thread", "value": "61910-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Test_2021_06_17.test_cal.test_cal"}]}